Index: socialFeed/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>HELP.md\r\n.gradle\r\nbuild/\r\n!gradle/wrapper/gradle-wrapper.jar\r\n!**/src/main/**/build/\r\n!**/src/test/**/build/\r\n\r\n### STS ###\r\n.apt_generated\r\n.classpath\r\n.factorypath\r\n.project\r\n.settings\r\n.springBeans\r\n.sts4-cache\r\nbin/\r\n!**/src/main/**/bin/\r\n!**/src/test/**/bin/\r\n\r\n### IntelliJ IDEA ###\r\n.idea/compiler.xml\r\n.idea/modules.xml\r\n.idea\r\n*.iws\r\n*.iml\r\n*.ipr\r\nout/\r\n!**/src/main/**/out/\r\n!**/src/test/**/out/\r\n\r\n### NetBeans ###\r\n/nbproject/private/\r\n/nbbuild/\r\n/dist/\r\n/nbdist/\r\n/.nb-gradle/\r\n\r\n### VS Code ###\r\n.vscode/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/socialFeed/.gitignore b/socialFeed/.gitignore
--- a/socialFeed/.gitignore	(revision 9984f4fbb4e00c9d33938b273de915894babe601)
+++ b/socialFeed/.gitignore	(date 1724386636351)
@@ -18,8 +18,6 @@
 !**/src/test/**/bin/
 
 ### IntelliJ IDEA ###
-.idea/compiler.xml
-.idea/modules.xml
 .idea
 *.iws
 *.iml
